_plugins/bidirectional_links_generator.rb

```diff
@@ -18,7 +18,7 @@ def generate(site)
         title_from_filename = File.basename(
```

Comment:
Given this variable will now point to a regular expression pattern, we could rename it:

```suggestion
        note_title_regexp_pattern = File.basename(
```